src/django/api/models/moderation_event.py

```
@@ -0,0 +1,122 @@
+import uuid
+from django.db import models
+from django.contrib.postgres import fields as postgres
```

Ruff

`django.contrib.postgres.fields` imported but unused

Remove unused import: `django.contrib.postgres.fields`

(F401)